src/content/notion.ts

```diff
@@ -75,6 +76,11 @@ export default class Notion {
     ];
   }
 
+  static buildDate(content: string | null): PropertyRequest<'date'> {
+    return { start: dayjs(content) };
```

Comment:
With the incoming value being a JavaScript `Date` object, we can use the `toISOString` function to convert to the desired string format for Notion. Also, since the incoming value can possibly be `null`, we'll want to check for that as well.

```suggestion
  static buildDate(date: Date | null): PropertyRequest<'date'> {
    return date && { start: date.toISOString() };
```